parmed/openmm/parameters.py

```diff
@@ -122,18 +138,87 @@ def write(self, dest, provenance=None):
+                   datetime.datetime.now().timetuple()[:3])
+        dest.write('  <Reference>%s</Reference>\n' % reference)
+        dest.write(' </Info>\n')
+
+    def _write_omm_atom_types(self, dest):
+        dest.write(' <AtomTypes>\n')
+        for (name, i), atom_type in iteritems(self.atom_types_tuple):
+            assert atom_type.atomic_number >= 0, 'Atomic number not set!'
+            element = Element[atom_type.atomic_number]
+            dest.write('  <Type name="%s" class="%d" element="%s" mass="%f"/>\n'
+                       % (name, i, element, atom_type.mass or Mass[element]))
+        dest.write(' </AtomTypes>\n')
+
+    def _write_omm_residues(self, dest):
+        self.typeify_templates()
+        dest.write(' <Residues>\n')
+        for name, residue in iteritems(self.residues):
+            if not isinstance(residue, ResidueTemplate):
+                continue
+            dest.write('  <Residue name="%s">\n' % residue.name)
+            for atom in residue.atoms:
+                dest.write('   <Atom name="%s" type="%s" charge="%f"/>\n' %
+                           (atom.name, atom.atom_type.number, atom.charge))
+            for bond in residue.bonds:
+                dest.write('   <Bond atomName1="%s" atomName2="%s" />\n' %
+                           (bond.atom1.name, bond.atom2.name))
+            if residue.head is not None:
+                dest.write('   <ExternalBond atomName="%s">\n' %
+                           residue.head.name)
+            if residue.tail is not None:
+                dest.write('   <ExternalBond atomName="%s">\n' %
+                           residue.tail.name)
+        dest.write(' </Residues>\n')
+
+    def _write_omm_bonds(self, dest):
+        if not self.bond_types: return
+        dest.write(' <HarmonicBondForce>\n')
+        bonds_done = set()
+        lconv = u.angstroms.conversion_factor_to(u.nanometers)
+        kconv = u.kilocalorie.conversion_factor_to(u.kilojoule) / lconv**2 * 2
+        for (a1, a2), bond in iteritems(self.bond_types):
+            if (a1, a2) in bonds_done: continue
+            bonds_done.add((a1, a2))
+            bonds_done.add((a2, a1))
+            dest.write('  <Bond class1="%s" class2="%s", length="%f", k="%f"/>\n'
+                       % (a1, a2, bond.req*lconv, bond.k*kconv))
+        dest.write(' </HarmonicBondForce>\n')
+
+    def _write_omm_angles(self, dest):
+        if not self.angle_types: return
+        dest.write(' <HarmonicAngleForce>\n')
+        angles_done = set()
+        tconv = u.degree.conversion_factor_to(u.radians)
+        kconv = u.kilocalorie.conversion_factor_to(u.kilojoule) * 2
+        for (a1, a2, a3), angle in iteritems(self.angle_types):
+            if (a1, a2, a3) in angles_done: continue
+            angles_done.add((a1, a2, a3))
+            angles_done.add((a3, a2, a1))
+            dest.write('  <Angle class1="%s" class2="%s" class3="%s" '
```

Comment:
We don't want to use `class` here---we want to use `type`, since we will be using the translation
* OpenMM `type` is AMBER atom type
* OpenMM `class` is AMBER Lennard-Jones class